R/visualize.R

```diff
@@ -0,0 +1,237 @@
+#' Fairness Accuracy Tradeoff Visualization
+#'
+#' @description
+#' This function specialize in comparing the Fairness vs Accuracy between learners through visualizations.
```

Comment:
Please run a spell check like grammarly. This has a lot of tiny errors

Reply:
Mark as resolved